proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,35 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+message UniquenessPolicy {
+  optional bool enabled = 1;
+}
+
+message UniquenessState {
+  bool has_collisions = 1;
+  repeated Collision collisions = 2;
+}
+
+message Collision {
+  string id = 1;
+  CollisionType type = 2;
+  string reference = 3;
+  repeated CollisionSource sources = 4;
+}
+
+message CollisionSource {
+  string id = 1;
+  string organization_id = 2;
+  string repository_id = 3;
+  string filepath = 4;
+}
+
+enum CollisionType {
+  COLLISION_TYPE_UNSPECIFIED = 0;
+  COLLISION_TYPE_FQN = 1;
+  COLLISION_TYPE_FILEPATH = 2;
+}
```

Comment:
@stefanvanburen I think this part of the change is probably the part that warrants some thought/feedback